x-pack/test/functional/apps/ml/data_frame_analytics/outlier_detection_creation.ts

```diff
@@ -5,19 +5,47 @@
  * 2.0.
  */
 
+import { TIME_RANGE_TYPE } from '@kbn/ml-plugin/public/application/components/custom_urls/custom_url_editor/constants';
 import type { FtrProviderContext } from '../../../ftr_provider_context';
 import type { AnalyticsTableRowDetails } from '../../../services/ml/data_frame_analytics_table';
 import type { FieldStatsType } from '../common/types';
+import {
+  type DiscoverUrlConfig,
+  type DashboardUrlConfig,
+  type OtherUrlConfig,
+} from '../../../services/ml/data_frame_analytics_edit';
+
+const testDiscoverCustomUrl: DiscoverUrlConfig = {
+  label: 'Show data',
+  indexPattern: 'ft_ihp_outlier',
```

Comment:
Does this data set have a time field? It would be nice for the tests on one of the types of jobs to use a time based index so that we could test the two different time range inputs.

Reply:
Added all tests for dataset with timefield in 518fabb4bc2b59e729c93231c9c258c0cbffa1b3